### ISSUE TITLE
Fixed channel issue on Qualcomm qcacld3.0

### ISSUE THREAD

**Issue by spiral009** [aanarchyy/bully#29]
I'm trying to run bully in a chroot environment on termux.
I can run airodump-ng on fixed channel with `--ignore-negative-one" option but I don't see such option in bully nor reaver , reaver just keeps segfault
```
root@localhost:~# bully   -b 08:40:F3:E9:D2:A1 wlan0
[!] Bully v1.4-00 - WPS vulnerability assessment utility
[P] Modified for pixiewps by AAnarchYY(aanarchyy@gmail.com)
[!] ioctl(SIOCGIWFREQ) on 'wlan0' failed with '0'
Segmentation fault

root@localhost:~# bully   -b 08:40:F3:E9:D2:A1 -c 4 wlan0
[!] Bully v1.4-00 - WPS vulnerability assessment utility
[P] Modified for pixiewps by AAnarchYY(aanarchyy@gmail.com)
[+] Switching interface 'wlan0' to channel '4'
[!] ioctl(SIOCSIWFREQ) on 'wlan0' failed with '-1'
[X] Unable to set channel on 'wlan0', exiting
```
I don't see any patch for fixed channel for qcacld3.0

**Comment by soxrok2212** [comment]
What is the chip you're trying to use?

**Comment by spiral009** [comment]
I'm not sure , my android is using snapdragon 870
https://github.com/kimocoder/qualcomm_android_monitor_mode

**Comment by soxrok2212** [comment]
This is probably a driver problem. Can you try with airodump and see if it is able to set a channel correctly?

**Comment by spiral009** [comment]
Well airodump-ng can capture on fixed channel with ```--ignore-negative-one``` option otherwise i get fixed channel -1 error

**Comment by soxrok2212** [comment]
Did you verify it actually listens on the correct channel when you use that option?

**Comment by spiral009** [comment]
Yeah ```airodump-ng wlan0 --bssid 08:40:F3:E9:D2:A1 -c 4 --ignore-negative-one```

![Screenshot_٢٠٢٣١٠١٠-٠٢٥٨٢١_Termux](https://github.com/aanarchyy/bully/assets/39057208/dfc8d69f-1a7f-4a21-aa19-ccdcc811b87b)
